package.json

```diff
@@ -36,6 +36,19 @@
     "@angular/router": "~14.2.7",
     "@asymmetrik/ngx-leaflet": "^14.0.1",
     "@asymmetrik/ngx-leaflet-markercluster": "^14.0.1",
+    "@capacitor-community/intercom": "^5.0.1",
+    "@capacitor/android": "^5.7.4",
+    "@capacitor/app": "^5.0.7",
+    "@capacitor/camera": "^5.0.9",
+    "@capacitor/cli": "^5.7.4",
+    "@capacitor/core": "^5.7.4",
+    "@capacitor/device": "^5.0.7",
+    "@capacitor/dialog": "^5.0.7",
+    "@capacitor/filesystem": "^5.2.1",
+    "@capacitor/geolocation": "^5.0.7",
+    "@capacitor/network": "^5.0.7",
+    "@capacitor/share": "^5.0.7",
+    "@capacitor/status-bar": "^5.0.7",
```

Comment:
Hi @ushahidlee I have a question. The new packages added in the `package.json`... What's their purpose in relation to what the PR/issue ticket is trying to resolve?